src/js/components/stock-movement-wizard/outbound/SendMovementPage.jsx

```diff
@@ -474,14 +503,29 @@ class SendMovementPage extends Component {
    */
   submitStockMovement(values) {
     this.props.showSpinner();
-    if (this.state.files.length) {
-      _.forEach(this.state.files, (file) => {
-        this.sendFile(file)
-          .then(() => Alert.success(this.props.translate('react.stockMovement.alert.fileSuccess.label', 'File uploaded successfuly!'), { timeout: 3000 }))
-          .catch(() => Alert.error(this.props.translate('react.stockMovement.alert.fileError.label', 'Error occured during file upload!')));
-      });
+    const { files } = this.state;
+    if (files.length > 1) {
+      this.sendFiles(files)
+        .then(() => {
+          Alert.success(this.props.translate('react.stockMovement.alert.fileSuccess.label', 'File uploaded successfuly!'), { timeout: 3000 });
```

Comment:
There is more than one file in this case so you can change label to "Files uploaded successfuly!"